masterlist.yaml

```
@@ -21621,7 +22074,9 @@ plugins:
         display: '[FileAccess Interface for Skyrim SE Scripts (FISSES)](https://www.nexusmods.com/skyrimspecialedition/mods/13956/)'
         condition: 'not checksum("DBM_RelicNotifications.esp", 663FAB6A) and not version("DBM_RelicNotifications.esp", "5.1.1", >=)'
     tag: [ Text ]
-    msg: [ *requiresMCM ]
+    msg: 
+      - *requiresMCM
+      - *patch3rdParty_SRPC
```

Make sure the aliases are in alphabetical order.

```suggestion
      - *patch3rdParty_SRPC
      - *requiresMCM
```